test/test_tensor_creation_ops.py

```diff
@@ -1046,6 +1047,8 @@ def test_float_to_int_conversion_finite(self, device, dtype):
         self._float_to_int_conversion_helper(vals, device, dtype, refs)
 
     # Note: CUDA will fail this test on most dtypes, often dramatically.
+    # NB: torch.uint16, torch.uint32, torch.uint64 excluded as this
+    # nondeterministically fails, warning "invalid value encountered in cast"
```

Comment:
Is that expected to be non-deterministic??

Reply:
I'm not sure, because I'm not exactly sure what this is testing. The statement that CUDA fails this dramatically seems to imply that what we're actually testing here might actually be UB.